lightning/src/events/mod.rs

```diff
@@ -164,7 +164,10 @@ pub enum ClosureReason {
 	///
 	/// [`ChannelMonitor`]: crate::chain::channelmonitor::ChannelMonitor
 	/// [`ChannelManager`]: crate::ln::channelmanager::ChannelManager
-	OutdatedChannelManager
+	OutdatedChannelManager,
+	/// The counterparty requested a cooperative close of a channel that had not established
+	/// a funding transaction yet. The channel has been immediately closed.
+	CounterpartyCoopClosedPrefundedChannel,
```

Comment:
Hmm... "prefunded" makes it sound like it was funded in advance. Shouldn't we say "unfunded"? Might be another occurrence of this predating the PR.

Reply:
Yeah exactly the reason I’ve started changing and avoiding “refunded”, but this was accidental, not preexisting 🤦‍♂️